### PR TITLE
Arshdeep added link

### PR DESCRIPTION
**Pull request by Arshdeep-0509**
profile added using json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/248"></a>

### CODE REVIEW

**Review by aayush89890 — CHANGES_REQUESTED** [review]
Hi @Arshdeep-0509 welcome to the community, but we don't make changes in ```package-lock.json``` & ```package.json``` files, please undo those changes and update your PR.

**Comment by Arshdeep-0509** [comment]
I have not made  any changes in  package-lock.json .

**Comment by CodingSpecies** [comment]
> Hi @Arshdeep-0509 welcome to the community, your json file is good but we don't make changes in `package-lock.json` & `package.json` files, please undo those changes and update your PR.

This seems to happen in other prs too @aayush89890 i think every time someone adds something it updates the version in the package lock.json. Thanks

File: public/data/Arshdeep-0509.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Arshdeep Singh",
+    "bio": "MERN stack developer || Student",
+    "avatar": "https://github.com/Arshdeep-0509.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Arshdeep-0509",
```
Comment (Panquesito7) [inline public/data/Arshdeep-0509.json:8]:
```suggestion
        "url": "https://github.com/Arshdeep-0509",
```

File: public/data/Arshdeep-0509.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        "icon": "github"
+      }
+     
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/Arshdeep-0509.json:14]:
```suggestion


```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by aayush89890** [comment]
> This seems to happen in other prs too @aayush89890 i think every time someone adds something it updates the version in the package lock.json. Thanks

That's correct @CodingSpecies but one should not commit those changes right?

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/248/conflicts). If you need any help with that, let us know in the [**Discord server**](http://discord.eddiehub.org).

**Comment by CodingSpecies** [comment]
> > This seems to happen in other prs too @aayush89890 i think every time someone adds something it updates the version in the package lock.json. Thanks
> 
> That's correct @CodingSpecies but one should not commit those changes right?

I think when they git add, they should only `git add profilename.json` instead of git add -A. thanks.

**Comment by aayush89890** [comment]
> I think when they git add, they should only git add profilename.json instead of git add -A. thanks.

yes that's correct @CodingSpecies

**Comment by eddiejaoude** [comment]
I will close this PR and manually add your profile data, feel free to update

**Pull request closed without merging**